contrib/clojure-package/examples/bert/src/bert/bert_sentence_classification.clj

```diff
@@ -157,4 +183,42 @@
 (comment
 
   (train (context/cpu 0) 3)
-  (m/save-checkpoint model {:prefix "fine-tune-sentence-bert" :epoch 3}))
+
+  (m/save-checkpoint model {:prefix fine-tuned-prefix :epoch 3})
+
+  
+  ;;;; Explore results from the fine-tuned model
+
+  ;; We need a predictor with a batch size of 1, so we can feed the
+  ;; model a single sentence pair.
+  (def fine-tuned-predictor
+    (infer/create-predictor (infer/model-factory fine-tuned-prefix
+                                                 [{:name "data0" :shape [1 seq-length] :dtype dtype/FLOAT32 :layout layout/NT}
+                                                  {:name "data1" :shape [1 seq-length] :dtype dtype/FLOAT32 :layout layout/NT}
+                                                  {:name "data2" :shape [1]            :dtype dtype/FLOAT32 :layout layout/N}])
+                            {:epoch 3}))
```

Comment:
Why do we need this hardcoded epoch number here?

Reply:
As I recall, we have to hard-code the epoch because otherwise we don't know which saved model to load from disk.

Reply:
With the `infer` API I suppose? Maybe we could change this at some point @kedarbellare?

Reply:
Responding to edit: `num-epoch` isn't in scope in the `comment`. I decided against defining it globally in order to parameterize a short REPL exploration. 

Another reason not to `def` the `3` here is that `num-epoch` is a value meant to be passed in from the command line, and the rich comment code is a parallel of invoking from the command line with that argument. So at a minimum we would need a new name.